nb5-proof/src/test/java/io/nosqlbench/nb5/proof/WorkloadContainerVerifications.java

```diff
@@ -0,0 +1,213 @@
+            };
+            logger.info("The final command line: " + String.join(" ", args));
+            ProcessResult runSchemaResult = invoker.run("run-workload", 30, args);
+
+
+            //STEP 2 Check runSchemaOut for errors
+            logger.info("Checking if the NB5 command resulted in any errors...");
+            assertThat(runSchemaResult.exception).isNull();
+            String runSchemaOut = String.join("\n", runSchemaResult.getStdoutData());
+            assertThat(runSchemaOut.toLowerCase()).doesNotContain("error");
+            logger.info("NB5 command completed with no errors");
+
+
+            //STEP 4 Check the cluster for created data
+            try (CqlSession session = CqlSession.builder().addContactPoint(new InetSocketAddress(hostIP, mappedPort9042)).withLocalDatacenter(datacenter).build()) {
+                //->Check for the creation of the keyspace baselines
+                logger.info("Checking for the creation of the keyspace \"baselines\"...");
+                ResultSet result = session.execute("SELECT keyspace_name FROM system_schema.keyspaces");
+                List<Row> rows = result.all();
+                boolean keyspaceFound = false;
+                for (Row row : rows) {
+                    if (row.getString("keyspace_name").equals("baselines")) {
+                        keyspaceFound = true;
+                        break;
+                    }
+                }
+                assertTrue(keyspaceFound);
+                logger.info("Keyspace \"baselines\" was found, nb5 command had created it successfully");
+
+                //->Check for the creation of the baselines keyvalue table
+                logger.info("Checking for the creation of the table \"baselines.keyvalue\"...");
+                result = session.execute("SELECT table_name FROM system_schema.tables WHERE keyspace_name='baselines'");
+                rows = result.all();
+                boolean tableFound = false;
+                for (Row row : rows) {
+                    if (row.getString("table_name").equals("keyvalue")) {
+                        tableFound = true;
+                        break;
+                    }
+                }
+                assertTrue(tableFound);
+                logger.info("Table \"baselines.keyvalue\" was found, nb5 command had created it successfully");
+
+                //->Check for the creation of the baselines keyvalue table
+                ResultSet resultSet = session.execute("SELECT count(*) FROM baselines.keyvalue");
+                Row row = resultSet.one();
+                long rowCount = row.getLong(0);
+                logger.info("Number of rows in baselines.keyvalue: " + rowCount);
+                assertTrue(rowCount >= 5);
+                logger.info("Table \"baselines.keyvalue\" has at least 5 rows of key-value pairs, nb5 command had created them successfully");
+
```

Comment:
This only matters for specific workloads. We should generalize it or disable it until we have a more cohesive approach for the workload variety.

Reply:
Code commented with more detail for reactivation later, maybe we can base this off the name of the test scenario? 'data-check-test' scenario could trigger this code for example, but a 'basic-check' would only establish connection and make sure the nb5 commands run with no errors?